cc/oak_containers_hello_world_trusted_app/BUILD

```diff
@@ -0,0 +1,89 @@
+#
```

Comment:
We also are currently trying to not include `oak_` prefix in directory names

And internally we already have a `cc/containers/` directory

So the example could be moved to `cc/containers/hello_world_trusted_app`

Reply:
Done.